setup.cfg

```diff
@@ -46,11 +46,11 @@ install_requires =
     zarr>=2.6.1
     natsort>=7.1.1
     pycromanager>=0.13.2
+    torch
```

Comment:
Not sure if we should pin a version(?)

Reply:
I don't think we need to yet. Unless you've had a fresh install fail for versioning reasons?

Reply:
There is no need to pin torch if we are only doing a fresh installation. But people may force it in some other environments where e.g. torch<1.8 is required, and that may bear minor differences from 2.0+ that we are testing against.

Reply:
Sounds good. @ziw-liu would you suggest `torch>=2.0.0`? That's what I've been testing against, and I just added.